src/ape/api/transactions.py

```diff
@@ -151,14 +151,21 @@ class ReceiptAPI(BaseInterfaceModel):
     logs: List[dict] = []
     status: int
     txn_hash: str
-    transaction: TransactionAPI
+    transaction: TransactionAPI | dict
```

Comment:
Not acceptable syntax <3.10

Also, I want this to be `TransactionAPI` only, so no union as a dict

Reply:
Damn linter I built auto fixed my original syntax. That's fixed now